doc/source/tutorials/advanced-consolidation.rst

```diff
@@ -0,0 +1,210 @@
+.. _advanced-consolidation:
+
+Advanced Consolidation
+======================
+
+In this section we explain consolidation in depth. Specifically,
+we describe the consolidation algorithm and all the configuration
+parameters that can be used to tune the consolidation performance
+to the user application.
+
+Fragment clean up
+-----------------
+
+Before the consolidation algorithm begins, TileDB applies a simple
+optimization in a pre-processing step, which may lead to great
+performance benefits depending on the "shape" of the existing
```

Comment:
Maybe clarify the performance benefits are for consolidation performance, not read performance.

Reply:
It is read performance as well. If you have 100 redundant fragments that a single fragment overwrites, deleting all 100 fragments and performing no further consolidation (because perhaps after this deletion step you end up with a single fragment) significantly boosts the reads.